onadata/settings/base.py

```diff
@@ -327,35 +326,34 @@ def skip_suspicious_operations(record):
 
 LOCALE_PATHS = [os.path.join(PROJECT_ROOT, 'locale'), ]
 
-DEBUG = os.environ.get('DJANGO_DEBUG', 'True') == 'True'
+DEBUG = env.bool('DJANGO_DEBUG', True)
 
 # Database (i.e. PostgreSQL)
 DATABASES = {
-    'default': dj_database_url.config(default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)
+    'default': env.db(default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)
 }
 # Replacement for TransactionMiddleware
 DATABASES['default']['ATOMIC_REQUESTS'] = True
 
-ALLOWED_HOSTS = os.environ.get('DJANGO_ALLOWED_HOSTS', '*').split(' ')
+ALLOWED_HOSTS = env.str('DJANGO_ALLOWED_HOSTS', '*').split(' ')
 
 # Domain must not exclude KPI when sharing sessions
-if os.environ.get('SESSION_COOKIE_DOMAIN'):
-    SESSION_COOKIE_DOMAIN = os.environ['SESSION_COOKIE_DOMAIN']
+SESSION_COOKIE_DOMAIN = env.str('SESSION_COOKIE_DOMAIN', None)
+if SESSION_COOKIE_DOMAIN:
     SESSION_COOKIE_NAME = 'kobonaut'
 
 SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
 
 # If not properly overridden, leave uninitialized so Django can set the default.
 # (see https://docs.djangoproject.com/en/1.8/ref/settings/#default-file-storage)
 if os.environ.get('KOBOCAT_DEFAULT_FILE_STORAGE'):
-    DEFAULT_FILE_STORAGE = os.environ.get('KOBOCAT_DEFAULT_FILE_STORAGE')
+    DEFAULT_FILE_STORAGE = env.str('KOBOCAT_DEFAULT_FILE_STORAGE')
 
-EMAIL_BACKEND = os.environ.get('EMAIL_BACKEND',
-                               'django.core.mail.backends.filebased.EmailBackend')
+EMAIL_BACKEND = env.str('EMAIL_BACKEND',
+                            'django.core.mail.backends.filebased.EmailBackend')
```

Comment:
Hmm, I would leave the indent the way it was. Otherwise, we could `Black`-format it to something like 
```python
EMAIL_BACKEND = env.str(
    'EMAIL_BACKEND', 'django.core.mail.backends.filebased.EmailBackend'
)
```

Reply:
I like the later format better. I'd be in favor of using black on the whole code base.

Reply:
So do I